modules/sbo-creating-a-postgresql-database-instance.adoc

```diff
@@ -77,10 +70,9 @@ The output, which takes a few minutes to display, verifies that the database is
 [source,terminal]
 ----
 NAME                               READY   STATUS    RESTARTS   AGE
-hippo-backup-nqjg-2rq94            1/1     Running   0          35s
-hippo-instance1-nw92-0             3/3     Running   0          112s
-hippo-pgbouncer-57b98f4476-znsk5   2/2     Running   0          112s
-hippo-repo-host-0                  1/1     Running   0          112s
+hippo-backup-9rxm-88rzq   0/1     Completed   0          2m2s
```

Comment:
I think you need to change the formatting for the preceding line, to get the columns to match up again:

```
NAME                               READY   STATUS    RESTARTS   AGE
hippo-backup-9rxm-88rzq   0/1     Completed   0          2m2s
hippo-instance1-6psd-0    4/4     Running     0          3m28s
hippo-repo-host-0         2/2     Running     0          3m28s
```

Reply:
Sure, addressed. PTAL. Thanks!